spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -406,12 +406,28 @@
             'documents-form-documents-field',
             'spec/fixtures/files/blank_job_spec.pdf'
           )
+
+          click_on 'Save and continue'
+
+          fill_in_application_details_form_fields(vacancy)
+          click_on 'Save and continue'
+
+          expect(page).to have_content(I18n.t('jobs.review'))
+          expect(page).to_not have_content('No documents uploaded')
+
+          verify_all_vacancy_details(vacancy)
+        end
+
+        scenario 'redirects to the vacancy review page when submitted successfully WITHOUT uploaded documents' do
```

Comment:
Can you turn the 'WITH' and 'WITHOUT' into `context`s please?

Reply:
The motivation here is neatness - prefer nesting of tests in the test output to eye-catching capital letters